roou.py

```diff
@@ -26,16 +21,17 @@
                  'This might raise an error if trying to add a SDSS PSF')
 
 # Path to PSF metadata (psFields) which the SDSS PSF tool uses as input.
-psfFields_dir = '/mnt/ds3lab/galaxian/source/sdss/dr12/psf-data'
-# Path to Hubble PSF if used:
+psfFields_dir = conf.core_path+'/source/sdss/dr12/psf-data'
 
-# The following paths are only needed, if the Hubble PSF is used for simulating 
+# Path to Hubble PSF if used:
+# The following paths are only needed, if the Hubble PSF is used for simulating
 # AGN point sources.
 path_to_Hubble_psf = '/mnt/ds3lab/dostark/hubble_psf_many_stars.fits'
 #path_to_Hubble_psf = '/mnt/ds3lab/dostark/PSFSTD_WFC3IR_F160W.fits'
-tmp_GALFIT = '/mnt/ds3lab/dostark/hubble_z_'+str(conf.redshift)+\
-             '/'+conf.filter_+'-band/tmp_for_GALFIT_'+str(conf.ext)+\
-             '_'+str(conf.stretch_type)+'_'+str(conf.scale_factor)+'/'
+
+# Directories for temporary files
+tmpdir_for_SExtractor = conf.stretch_setup+'/tmp_for_SExtractor/'
+tmp_GALFIT = conf.stretch_setup+'/tmp_for_GALFIT/'
```

Comment:
@dominicstark Is it a problem to change the paths of these temporary directories like that ? Information is the same, just organized in a different way